Formula/t/topfew.rb

```diff
@@ -0,0 +1,18 @@
+class Topfew < Formula
+  desc "Finds the field values which appear most often in a stream of records"
+  homepage "https://github.com/timbray/topfew"
+  url "https://github.com/timbray/topfew/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "ac277366c1f12e88b7a050a11d8eb35b1b518b1b94b0ec58a34ff935f76d9269"
+  license "GPL-3.0-or-later"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-trimpath", "-o", "topfew"
+    bin.install "topfew" => "tf"
+  end
+
+  test do
+    assert_match  "1 bar", shell_output("echo 'foo bar' | #{bin}/tf -f 2")
```

Comment:
```suggestion
    assert_match  "1 bar", pipe_output("#{bin}/tf -f 2", "foo bar")
```

Reply:
Cool thanks will do.  A bit grumpy because I ran all the recommended "brew audit" variations from the cookbook and none of them ran the actual test where I'd fat-fingered the expected result.

Reply:
OK, I'm going to have to get some help from a brew expert. I'm making changes to `/opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/t/topfew.rb` and doing a `brew reinstall --build-from-source --verbose --debug topfew` and the changes don't affect the results of "brew audit" or "brew test", so I'm missing a step.  I know who to ask, will return once I've sorted this out.

Reply:
The problem was that the build step was failing, it didn't like ` *std_go_args(out: bin/"tf")`.  Having fixed that, I discovered that `pipe_output("#{bin}/tf -f 2", "foo bar")` also doesn't work, while my original  formulation, `assert_match "1 bar", shell_output("echo 'foo bar' | #{bin}/tf -f 2")` does work.  Are we perhaps using a different version of brew?